frontend/src/core/loaders/components/SkeletonLoader.css

```diff
@@ -0,0 +1,33 @@
+.skeleton-loader p {
+    height: 0.875rem;
+    background-color: #E2E2E2;
+    animation: fading 1.5s infinite;
+    overflow: hidden;
```

Comment:
Not sure this is needed here. But you should add a new rule on top to prevent scrollbar on the loader:

```
.skeleton-loader.entities {
    height: 100%;
    overflow: hidden;
}
```